runtime/src/comm/ofi/comm-ofi-oob-slurm-pmi2.c

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ * 
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * 
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+// PMI2-based out-of-band support for the OFI-based Chapel comm layer.
+//
+
+#include "chplrt.h"
+#include "chpl-env-gen.h"
+
+#include "chpl-comm.h"
+#include "chpl-mem.h"
+#include "chpl-mem-sys.h"
+#include "chpl-gen-includes.h"
+#include "chplsys.h"
+#include "error.h"
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+
+#include "comm-ofi-internal.h"
+
+
+#define PMI2_SUCCESS 0
+#define PMI2_ID_NULL -1
+
+#define PMI2_MAX_VALLEN 1024
```

Comment:
Do we want to make an issue for this being hardcoded? Or do we think this is ok?

Reply:
Yeah, I'll make an issue.